src/i18n/keymap_extras/convert_keymap_extras_header.js

```diff
@@ -0,0 +1,300 @@
+  ['S(KC_QUOT)', 'KC_DQUO']
+]);
+
+const intlAliasPattern = '\\w+';
+const macroExpansionPattern = '[\\w\\(\\)]+';
+// It's important that `keysymPattern` can match whitespace characters that are not
+// ASCII 0x20 <SPACE> like 0x00a0 <NON-BREAKING SPACE>.
+const keysymPattern = '[^ ]+';
+const commentPattern = '\\s+\\(.+\\)';
+const kcAliasDefRegExp = new RegExp(
+  `# *define +(${intlAliasPattern}) +(${macroExpansionPattern}) *// *(${keysymPattern})(${commentPattern})?`
+);
+
+function extractBasicKc(kc) {
+  if (!kc.includes('(')) {
+    // Not a modded keycode string.
+    return kc;
+  }
+  return kc.slice(kc.lastIndexOf('(') + 1, kc.indexOf(')')).trim();
+}
+
+function translateToUS(kc, intl2us) {
+  const basicKc = extractBasicKc(kc);
+  const basicUSKc = intl2us.get(basicKc) || basicKc;
+  if (!basicUSKc.includes('KC_')) {
+    console.error('State of intl2us:');
+    console.error(intl2us);
+    if (!intl2us.has(basicKc)) {
+      throw new Error(`US translation of "${basicKc}" not found in intl2us!`);
+    } else {
+      throw new Error(
+        `${basicKc} gets translated as ${basicUSKc}, in intl2us, but this is not actually a valid US keycode!`
+      );
+    }
+  }
+  return kc.replace(basicKc, basicUSKc);
+}
+
+function computeIntl2US(lines) {
+  let intl2us = new Map();
+  for (const aliasDefinition of lines.filter((line) =>
+    kcAliasDefRegExp.test(line)
+  )) {
+    const [fullMatch, intlAlias, macroExpansion] =
+      kcAliasDefRegExp.exec(aliasDefinition);
+    const usAlias = translateToUS(macroExpansion, intl2us);
+    intl2us.set(intlAlias, usAlias);
+  }
+  return intl2us;
+}
+
+function computeKcInfo(lines, intl2us) {
+  let kcInfo = new Map();
+  for (const aliasDefinition of lines.filter((line) =>
+    kcAliasDefRegExp.test(line)
+  )) {
+    const [fullMatch, intlAlias, macroExpansion, keysym, comment] =
+      kcAliasDefRegExp.exec(aliasDefinition);
+    let readableKeysym = keysym;
+    if (keysym.length === 1 && invisibleChar2readableName.has(keysym)) {
+      readableKeysym = invisibleChar2readableName.get(keysym);
+    } else if (/\p{gc=Mn}/u.test(keysym)) {
+      // This obscure-looking regular expression is used to detect unicode characters
+      // pertaining to the [Marking, non-spacing](https://www.fileformat.info/info/unicode/category/Mn/list.htm) general category.
+      // This category includes characters such as COMBINING HORN which can, surprisingly enough, combine with the (double) quote
+      // character used to start a string in the generated output, and thus ruin the entire syntax.
+      //
+      // For an example of a keymap_extras header containing those characters, check out keymap_bepo.h
+      const dottedCircle = '\u25cc';
+      readableKeysym = dottedCircle + keysym;
+    }
+    // aka the hover tip
+    const title =
+      intlAlias + (comment === undefined ? '' : ' ' + comment.trim());
+    kcInfo.set(translateToUS(macroExpansion, intl2us), {
+      intlAlias: intlAlias,
```

Comment:
you can just use `intlAlias` and it will include it as part of object creating syntax. same goes for comment, and title.

Reply:
Solved in https://github.com/qmk/qmk_configurator/pull/1161/commits/16ee0e1116b6aed69f0365e5b47234cb5570892e